examples/AWS IoT Example/aws_mqtt_config_example.ino

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017, Helium Systems, Inc.
+ * All Rights Reserved. See LICENCE.txt for license information
+ */
+
+#include "Arduino.h"
+#include "Board.h"
+#include "Helium.h"
+#include "HeliumUtil.h"
+#include "ArduinoJson.h"
+
+// NOTE: This sample uses the "AWS IoT" channel which is available
+// by in the Helium Dashboard. To use another channel, change the channel
+// name to the target channel and adjust the Config variable name to match
+// the cloud service's variable name
+//
+// ***AWS IoT*** Once the application is running on the device, change the global
+// config variable "interval" in the Device Shadow.
+//
+// After one previous sleep cycle you should see the message arrive at
+// the new interval rate (in milliseconds)
+//
+#define CHANNEL_NAME "YOUR CHANNEL NAME HERE"
+#define CONFIG_INTERVAL_KEY "channel.interval"
```

Comment:
Could you use channel.interval_ms to indicate millis or channel.interval_s to indicate seconds? That way it's a bit clearer how long an interval is